plugins/com.google.cloud.tools.eclipse.util/src/com/google/cloud/tools/eclipse/util/io/HttpUtil.java

```diff
@@ -31,6 +41,37 @@
   public static final int DEFAULT_CONNECT_TIMEOUT_MS = 3000;
   public static final int DEFAULT_READ_TIMEOUT_MS = 3000;
 
+  private static final String MULTIPART_BOUNDARY =
```

Comment:
Theoretically, we should ensure parameter values don't have this particular boundary value, but the chance that there will be an overlap is basically 0 in practice, so I think just using this boundary is an acceptable compromise.